gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -391,7 +447,7 @@ public static RegionFilter equals(RegionField field, String value) {
     }
 
     /**
-     * Returns an equality filter for the given field and string value. For string fields,
+     * Returns a not-equals filter for the given field and string value. For string fields,
      * {@code value} is interpreted as a regular expression using RE2 syntax. {@code value} must
```

Comment:
Fine as is. linking RE2 to https://github.com/google/re2/wiki/Syntax would be nice.